src/main/java/frc/robot/commands/States/prep_coral/PrepCoralLv.java

```diff
@@ -24,22 +24,21 @@ public class PrepCoralLv extends Command {
   Motion globalMotion;
   Rotors globalRotors;
   StateMachine globalStateMachine;
-  Distance globalHeight;
   Pose2d closestPoseByRotation;
   MechanismPositionGroup prepL2;
   MechanismPositionGroup prepL3;
   MechanismPositionGroup prepL4;
+  int targetLevel = 0; // Placeholder for target height, adjust as needed
 
   public PrepCoralLv(StateMachine globalStateMachine, Motion subMotion, Rotors subRotors, Drivetrain subDrivetrain,
-      Distance height) {
+      Distance height, int level) {
```

Comment:
```suggestion
      int level) {
```
Also need to update StateMachine